### DISCUSSION TITLE
CTkmessagebox Gives me a wierd error

### DISCUSSION THREAD

**Discussion by aayushkdev** [Akascape/CTkMessagebox#41] [Q&A]
So I have used this message box in one or two places in my code and this error occurs on random intervals while using my app, where I wouldn't even have the message box called.

![Screenshot 2023-11-06 175829](https://github.com/Akascape/CTkMessagebox/assets/69899544/9f01c53b-1a3e-486c-b980-80e1f8ba86f6)

**Comment by Akascape** [comment]
@lyr-ast Are you using the latest version?

**Reply by aayushkdev** [reply]
Yep, I am, version 2.5

**Reply by Akascape** [reply]
@lyr-ast Can you give the code example?

**Reply by aayushkdev** [reply]
I believe I have identified what I was doing wrong. I had a function that deletes items from a database and displays a message box when the deletion is successfully completed. When I attempted to delete numerous records one by one which triggered the message box after each deletion and after deleting a few records this error occurred. I didn't notice it at first, so I thought it was appearing randomly.

You might be able to reproduce this error by creating a function with a similar message box and calling it multiple times using a button.
